owncloudApp/src/test/java/com/owncloud/android/presentation/viewmodels/settings/SettingsAdvancedViewModelTest.kt

```diff
@@ -0,0 +1,57 @@
+package com.owncloud.android.presentation.viewmodels.settings
+
+import com.owncloud.android.data.preferences.datasources.SharedPreferencesProvider
+import com.owncloud.android.presentation.ui.settings.fragments.SettingsAdvancedFragment.Companion.PREF_SHOW_HIDDEN_FILES
+import com.owncloud.android.presentation.viewmodels.ViewModelTest
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.verify
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import org.junit.After
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Test
+
+@ExperimentalCoroutinesApi
+class SettingsAdvancedViewModelTest : ViewModelTest() {
```

Comment:
We use mainly the ViewModelTest to switch between dispatchers and test livedatas, in this case, we can get rid of it 👍

Reply:
done